pkg/services/accesscontrol/models.go

```diff
@@ -341,16 +341,40 @@ const (
 
 	// Dashboard scopes
 	ScopeDashboardsAll = "dashboards:*"
+
+	// Alerting rules actions
+	ActionAlertingRuleCreate = "alert.rules:create"
+	ActionAlertingRuleRead   = "alert.rules:read"
+	ActionAlertingRuleUpdate = "alert.rules:update"
+	ActionAlertingRuleDelete = "alert.rules:delete"
+
+	// Alerting instances (+silences) actions
+	ActionAlertingInstanceCreate = "alert.instances:create"
+	ActionAlertingInstanceUpdate = "alert.instances:update"
+	ActionAlertingInstanceRead   = "alert.instances:read"
+
+	// Alerting Notification policies actions
+	ActionAlertingNotificationsCreate = "alert.notifications:create"
+	ActionAlertingNotificationsRead   = "alert.notifications:read"
+	ActionAlertingNotificationsUpdate = "alert.notifications:update"
+	ActionAlertingNotificationsDelete = "alert.notifications:delete"
+
+	// External alerting rule actions
```

Comment:
```suggestion
	// External alerting rule actions. We can only narrow it down to writes or reads, as we don't control the atomicity in the external system.
```

Reply:
All suggestions are addressed in https://github.com/grafana/grafana/pull/46553